.env

```diff
@@ -0,0 +1,10 @@
+# Please note that this is an example .env file, and you should
```

Comment:
I removed configuration from the source code and put it into an `.env` file.  This file is then automatically loaded.  It also allows us to overwrite any particular environment variable, simply by setting it before we run anything. This behavior is also honored by docker, so it should propagate all the way down.